plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -706,13 +706,11 @@ end
 
 function OPDSBrowser:browseSearchable(browse_url, username, password)
     self.search_server_dialog = InputDialog:new{
-        title = _("Search OPDS catalog"),
+        title = _("Enter author or title to search for"),
```

Comment:
A bit reductive, you can arguably match on tags in a Calibre catalog.

(i.e. , I prefer the previous title + hint combo, possibly with minor tweaks).

Reply:
> I prefer the previous title + hint combo

Do you mean `input_hint`?
There is just a `hint` in the code
https://github.com/koreader/koreader/blob/6d6760c2b3530b1469f127e6c06a3b49b3e1eca5/plugins/opds.koplugin/opdsbrowser.lua#L708
but I cannot find it in the InputDialog so I've removed it.

Reply:
Possibly there was a typo in the field name, yeah, but I know the feature exists ;p.